src/bootstrap/lib.rs

```diff
@@ -128,25 +128,26 @@ use filetime::FileTime;
 
 use crate::util::{exe, libdir, CiEnv};
 
+mod builder;
+mod cache;
 mod cc_detect;
 mod channel;
 mod check;
-mod test;
 mod clean;
 mod compile;
-mod metadata;
 mod config;
 mod dist;
 mod doc;
 mod flags;
+mod format;
 mod install;
+mod metadata;
 mod native;
 mod sanity;
-pub mod util;
-mod builder;
-mod cache;
+mod test;
 mod tool;
 mod toolstate;
+pub mod util;
```

Comment:
It'd be great to remove these changes (I think none of them should be necessary, possibly modulo the pub here, though I'm not sure that it's for either)